src/main/java/Tasks/TaskList.java

```diff
@@ -32,37 +32,41 @@ public void addTask(Task task) {
      *
      * @param itemNum is the index of the task to be deleted
      */
-    public void deleteTask(int itemNum) {
+    public String deleteTask(int itemNum) {
         assert(itemNum>=0 && itemNum<taskList.size()):"itemNum does not fall within range";
-        System.out.println("Noted. I've removed this task:");
-        System.out.println("  " + taskList.get(itemNum));
+        String toReturn = "";
```

Comment:
I feel like the string building should occur in the Ui class instead of the TaskList class?